ZXing.Net.MAUI/Platforms/Android/CameraManager.android.cs

```diff
@@ -71,9 +74,64 @@ public void Connect()
 
 				UpdateCamera();
 
-			}), ContextCompat.GetMainExecutor(Context.Context)); //GetMainExecutor: returns an Executor that runs on the main thread.
+				AutoFocus();
+				setupAutoFocusTimer();
```

Comment:
This seems like it adds a continuous autofocus that's never canceled, even when you explicitly call one of the other functions.